integrations/client/test_delphi_epidata.py

```diff
@@ -284,21 +284,28 @@ def test_covidcast(self):
       # check result
       self.assertEqual(response_1, {'message': 'no results', 'result': -2})
 
-  @patch('requests.post')
-  @patch('requests.get')
-  def test_request_method(self, get, post):
+  @patch('requests_cache.CachedSession')
+  @patch('requests.Session')
+  def test_request_method(self, _Session, _CachedSession):
     """Test that a GET request is default and POST is used if a 414 is returned."""
-    with self.subTest(name='get request'):
+    with self.subTest(name='get request, no cache'):
+      Session = MagicMock()
+      _Session.return_value = Session
       Epidata.covidcast('src', 'sig', 'day', 'county', 20200414, '01234')
-      get.assert_called_once()
-      post.assert_not_called()
+      Session.request.assert_called_once()
```

Comment:
I'd use the assert_called_once_with() functions here to check the post vs get calls

Reply:
Do you know if there's a way to check only one of the arguments win `assert_called_once_with`? I didn't want to write out the full params dict, but maybe I'm just being lazy.

Reply:
You could try accessing the mocks call_args_list attribute and checking for inclusion of an element